(Update)Additional Documentation for Conditionals (#7089)
harness/developer-hub: docs/platform/pipelines/step-skip-condition-settings.md

2. **Step group:** Step group-level conditional execution settings determine when a step group should run. If triggered, the steps in the group start running.
3. **Step:** Step-level conditional execution settings determine if individual steps should run. Step conditions are evaluated after stage and step group conditions. This means that step-level conditional execution settings effectively override the stage and step group conditional execution settings. For example, if a stage-level condition evaluates to true and causes a stage to run, each individual step-level condition within that stage is evaluated separately to determine if each individual step should run.

> ### Unless Failure Strategy is Specified, JEXL Booleans will result in a Skipped Stage
> For users setting a JEXL Boolean condition, it is important to remember that a conditional that evaluates false will result in a skipped step or stage.  
> 
> This will still show a step or stage as completing successfully.
> ![](./static/conditional-skipped.png)
> 
> Depending on the situation, this may be the desired effect.  Otherwise, users should [implement a failure strategy for the step/stage](https://developer.harness.io/docs/continuous-delivery/x-platform-cd-features/executions/step-and-stage-failure-strategy).
> 

Reviewer: ```suggestion
### Important note

For users setting a JEXL Boolean condition, it is important to remember that a conditional that evaluates false will result in a skipped step or stage.  

This will still show a step or stage as completed successfully.
![](./static/conditional-skipped.png)

Depending on the situation, this might be the desired effect. Otherwise, you should [implement a failure strategy for the step/stage](/docs/continuous-delivery/x-platform-cd-features/executions/step-and-stage-failure-strategy).
```
